server/paymentProviders/privacy/webhook.ts

```diff
@@ -0,0 +1,43 @@
+import { Request } from 'express';
+
+import logger from '../../lib/logger';
+import * as privacyLib from '../../lib/privacy';
+import models from '../../models';
+import { Transaction } from '../../types/privacy';
+
+import privacy from './index';
+
+async function webhook(req: Request & { body: Transaction; rawBody: string }): Promise<void> {
+  const virtualCard = await models.VirtualCard.findOne({
+    where: {
+      id: req.body.card.token,
+    },
+    include: [
+      { association: 'collective', required: true },
+      { association: 'host', required: true },
+    ],
+  });
+  if (!virtualCard) {
+    logger.error('privacy/webhook: could not find VirtualCard', { body: req.body });
+    return;
+  }
+
+  const host = virtualCard.host;
+  const collective = virtualCard.collective;
+  const [connectedAccount] = await host.getConnectedAccounts({
+    where: { service: 'privacy', deletedAt: null },
```

Comment:
:ice_cream: Unless I'm mistaking, `deletedAt: null` is always included unless you specify `paranoid: false`

```suggestion
    where: { service: 'privacy' },
```